meilisearch/_httprequests.py

```diff
@@ -22,6 +22,7 @@ def __init__(self, config: Config) -> None:
             "Authorization": f"Bearer {self.config.api_key}",
             "User-Agent": _build_user_agent(config.client_agents),
         }
+        self.session = requests.Session()
```

Comment:
And add `session: Session` to the `__init__` parameters.

```suggestion
        self.session = session
```